pom.xml

```diff
@@ -355,7 +355,7 @@
         <carbon.identity.account.association.version>5.3.7</carbon.identity.account.association.version>
         <maven.findbugsplugin.version>3.0.5</maven.findbugsplugin.version>
         <carbon.business-process.version>4.5.2</carbon.business-process.version>
-        <identity.oauth.version>6.4.150</identity.oauth.version>
+        <identity.oauth.version>6.6.34-SNAPSHOT</identity.oauth.version>
```

Comment:
make sure to update the released version when the oauth PR is merged

Reply:
Addressed in [81a8d70](https://github.com/wso2/identity-api-user/pull/129/commits/81a8d7074753b74f8bb43429db8d4ff80383675f)